Microservices/user-service/src/main/java/app/sportahub/userservice/service/user/UserServiceImpl.java

```diff
@@ -1,5 +1,15 @@
 package app.sportahub.userservice.service.user;
 
+import java.sql.Timestamp;
```

Comment:
Can you fix the imports in this file? Use `Ctrl + Shift + L` to reformat the file